src/pages/Profile.jsx

```diff
@@ -1,9 +1,12 @@
 import React from "react";
+import Box from "@material-ui/core/Box";
 
 const Profile = () => (
-  <div className="Profile" data-test="Profile-page">
-    <h1>Profile</h1>
-  </div>
+  <Box mt="40" mx="auto ">
+    <div className="Profile" data-test="Profile-page">
+      <h1>Profile</h1>
```

Comment:
🔧 add test for the heading

Reply:
This component no longer contains this element per latest commit:  c0dd94230ccd7f24127e61d1966455951fa0dd42